README.md

```diff
@@ -81,19 +82,19 @@ Migrating from 1.0 to 2.0
 
 ### Configuration
 
-The `placeholdertitle` property no longer exists, it has been replaced by `placeholder-info` which accepts an array containing one or more of `'title'`, `'description'`, `'brand'`, `'duration'`.
+The `placeholdertitle` property no longer exists, it has been replaced by `placeholder-info` which accepts an array containing one or more of `'title'`, `'description'`, `'brand'`.
 
 ```diff
 <div class="video-container">
-    <div class="o-video" data-o-component="o-video"
-        data-o-video-source="Brightcove"
-        data-o-component="o-video"
-        data-o-video-id="4165329773001"
-        data-o-video-advertising="true"
-        data-o-video-placeholder="true"
--        data-o-video-placeholdertitle="true"
-+        data-o-video-placeholder-info="['title']"
-    ></div>
+	<div class="o-video" data-o-component="o-video"
```

Comment:
This is because it is using tabs instead of spaces.
You can change how big a tab shows on github using `?ts=2`, where `2` is the number of spaces used to represent a tab.

https://github.com/Financial-Times/o-video/pull/53/files?ts=2